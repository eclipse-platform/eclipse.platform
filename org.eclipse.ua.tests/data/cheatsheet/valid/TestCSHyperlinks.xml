<?xml version="1.0" encoding="UTF-8" ?>

<!--
     Copyright (c) 2020 1C-Soft LLC and others.

     This program and the accompanying materials
     are made available under the terms of the Eclipse Public License 2.0
     which accompanies this distribution, and is available at
     https://www.eclipse.org/legal/epl-2.0/

     SPDX-License-Identifier: EPL-2.0

     Contributors:
         George Suaridze <suag@1c.ru> (1C-Soft LLC) - Bug 559885
 -->
<cheatsheet title="Hyperlinks Demo">
	<intro>
		<description>
			Welcome to the hyperlinks cheat sheet demo.
			<br/>
			Click the <a href="http://org.eclipse.ui.cheatsheet/execute?command=org.eclipse.search.ui.openSearchDialog">link</a> to open <b>Search</b> dialog.
			<br/>
			Open <a href="http://org.eclipse.ui.cheatsheet/execute?command=org.eclipse.ui.newWizard%28newWizardId%3Dorg.eclipse.ui.wizards.new.project%29">new project</a> wizard.
			<br/>
			Activate <a href="http://org.eclipse.ui.cheatsheet/showView?id=org.eclipse.pde.runtime.LogView">Error Log</a> view.
		</description>
	</intro>
	<item
		title="Step 1">
		<description>
			Open <a href="https://eclipse.org">eclipse.org</a> in external browser.	
		</description>
	</item>
</cheatsheet>
