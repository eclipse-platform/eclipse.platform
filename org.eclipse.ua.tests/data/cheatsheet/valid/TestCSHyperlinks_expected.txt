<?xml version="1.0" encoding="UTF-8" ?>
<cheatsheet
      title="Hyperlinks Demo">
   <item
         title="Introduction"
         description="<form><p>Welcome to the hyperlinks cheat sheet demo.
   <br/>Click the <a href="http://org.eclipse.ui.cheatsheet/execute?command=org.eclipse.search.ui.openSearchDialog">link</a> to open <b>Search</b> dialog.
   <br/>Open <a href="http://org.eclipse.ui.cheatsheet/execute?command=org.eclipse.ui.newWizard%28newWizardId%3Dorg.eclipse.ui.wizards.new.project%29">new project</a> wizard.
   <br/>Activate <a href="http://org.eclipse.ui.cheatsheet/showView?id=org.eclipse.pde.runtime.LogView">Error Log</a> view.
  </p></form>"
         Href="null"
         contextId="null">
      <nullAction/>
      <nullList/>
      <nullPerformWhen/>
      <nullList/>
   </item>
   <list>
      <item
            title="Step 1"
            description="<form><p>Open <a href="https://eclipse.org">eclipse.org</a> in external browser. 
  </p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
   </list>
</cheatsheet>
